src/state/trajectory/conversion-data-types.ts

```diff
@@ -0,0 +1,40 @@
+export enum AvailableEngines {
+    Cytosim = "cytosim",
+    CellPack = "cellPACK",
+    Smoldyn = "Smoldyn",
+    SpringSalad = "SpringSaLaD",
+}
+
+export interface CustomType {
+    "python::module": string;
+    "python::object": string;
```

Comment:
I can't remember why these should be named `python::module` when they could be a more js friendly `pythonModule` or `pythonModuleName`

Reply:
I'm not sure, but this is the convention throughout the ui-templates in simulariumio. It seems like we would want our typing to match the templates, whatever we go with.